R/normalize.R

```diff
@@ -122,6 +122,7 @@ prep.step_normalize <- function(x, training, info = NULL, ...) {
     ))
 
     sds <- sds[-which_sd]
+    means <- means[-which_sd]
```

Comment:
I don't think that this should change. Just because the data cannot be scaled, we can still center them.